### PR TITLE
Create Github issue on new redmine releases

### PR DESCRIPTION
**Pull request by Art4**
This PR uses the [RSS to Issues GitHub Action](https://github.com/marketplace/actions/rss-to-issues) to check the [Redmine news RSS feed](https://www.redmine.org/projects/redmine/news.atom) every 24h for new Redmine releases. If a new post is published, a Github issue will be created. See [here](https://github.com/Art4/php-redmine-api/issues/1) and [here](https://github.com/Art4/php-redmine-api/issues/2) as example.

This way we will get notified if a new Redmine version is released.

### CODE REVIEW

File: .github/workflows/check-for-releases.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Check for new Redmine releases
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Run this Action every day a 5:35am UTC
```
Comment (kbsali) [inline .github/workflows/check-for-releases.yml]:
```suggestion
    # Run this Action every day a 5:37am UTC
```
:sweat_smile:

File: .github/workflows/check-for-releases.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check for updates
+        uses: git-for-windows/rss-to-issues@v0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          feed: https://www.redmine.org/projects/redmine/news.atom
+          prefix: "Update the Behat tests: "
+          #character-limit: 255
```
Comment (kbsali) [inline .github/workflows/check-for-releases.yml:21]: needed?

**Review by kbsali — APPROVED** [review]
wow! very cool idea! :heart_eyes:

File: .github/workflows/check-for-releases.yml (hunk 2 above)
Comment (Art4) [reply .github/workflows/check-for-releases.yml:21]: I've tested the limit with a dry-run, see https://github.com/Art4/php-redmine-api/actions/runs/9681576974/job/26712484988

The content of the issue would be:

```
Redmine [5.1.3](https://www.redmine.org/versions/195) and [5.0.9](https://www.redmine.org/versions/194) have been released and are now available for [download](https://www.redmine.org/projects/redmine/wiki/Download). These are maintenance releases and onl…

---
## Would you like to know more?
Read the full article on the following website:

https://www.redmine.org/news/145
```

I think the full content would be nicer, so I have commented out the limit.

**Comment by kbsali** [comment]
up to you, i think that for a starter it's great already!

**Pull request merged**